test/model.jl

```diff
@@ -1,3 +1,17 @@
+@testset "serialization" begin
+    (; model_def, data) = JuliaBUGS.BUGSExamples.rats
+    model = compile(model_def, data)
+    serialize("m.jls", model)
+    deserialized = deserialize("m.jls")
+    @testset "test values are correctly restored" begin
+        for vn in MetaGraphsNext.labels(model.g)
+            @test isequal(
+                get(model.evaluation_env, vn), get(deserialized.evaluation_env, vn)
+            )
+        end
+    end
```

Comment:
Would checking that other fields of the model are preserved also be useful?

Reply:
yeah, make sense, have done this